pk3DS/Subforms/Gen7/GameSummary7.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Forms;
+using pk3DS.Core;
+using pk3DS.Core.Structures;
+using pk3DS.Core.Structures.PersonalInfo;
+
+namespace pk3DS
+{
+    public partial class GameSummary7 : Form
+    {
+        public GameSummary7()
+        {
+            InitializeComponent();
+            Setup();
+        }
+        #region Global Variables
+
+        private readonly List<PersonalInfo> pkms = Main.Config.Personal.Table.ToList();
+        private readonly List<Move> moves = Main.Config.Moves.ToList();
+        private readonly List<int> types = Main.Config.Personal.Table.SelectMany(pkm => pkm.Types).ToHashSet().ToList();
```

Comment:
Does the ToHashSet provide any value if you're turning it to a list immediately after?